src/scenes/Home/ParticipantHomeView/WelcomeBanner.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+import { Box, Grid, Typography, useTheme } from '@mui/material';
+
+import DialogCloseButton from 'src/components/common/DialogCloseButton';
+import strings from 'src/strings';
+
+const WelcomeBanner = () => {
+  const theme = useTheme();
+
+  const handleOnClick = () => {
+    // tslint:disable:no-console
+    console.log('close');
+  };
```

Comment:
Does this still need to be wired up to close the dialog?

Reply:
That is in the subsequent PR